docs/roadmap.md

```diff
@@ -14,26 +14,151 @@ ideas and plans with you.
 
 Below are the prioritised roadmap items:
 
-1. **[In Preview](./configuration/identity-providers/oidc.md)** *this roadmap item is in preview status, more
-   information can be found in the docs*. 
-   [Authelia acts as an OpenID Connect Provider](https://github.com/authelia/authelia/issues/189). This is a high
-priority because currently the only way to pass authentication information back to the protected app is through the
-use of HTTP headers as described
-[here](https://www.authelia.com/docs/deployment/supported-proxies/#how-can-the-backend-be-aware-of-the-authenticated-users)
-however, many apps either do not support this method or are starting to move away from this in favour of OpenID Connect or OAuth2
-internally or via plugins.
-
-2. [Administration interface](https://github.com/authelia/authelia/issues/974). This is useful in many cases to
+## 1. OpenID Connect
```

Comment:
I think it would be clearer to have the content of OIDC item (the details below) in a dedicated page under the roadmap one in order to keep visibility on the overall roadmap. You can keep one or two lines summarizing the item and add a link to the detailed page underneath. WDYT?

Reply:
Sounds fine to me! I'll reopen with that change and just merge it.